src/OrchardCore.Themes/TheTheme/Views/MenuItemLink-ContentMenuItem.cshtml

```diff
@@ -6,17 +6,16 @@
     var routeValues = new RouteValueDictionary(AutorouteOptions.Value.GlobalRouteValues);
     routeValues[AutorouteOptions.Value.ContentItemIdKey] = menuItemPart.ContentItem.Content.ContentMenuItemPart.SelectedContentItem.ContentItemIds[0];
     var linkUrl = Url.RouteUrl(routeValues);
-}
 
-@if ((int)Model.Level > 0)
-{
-    <a class="dropdown-item" href="@linkUrl">@menuItemPart.Name</a>
-}
-else if (Model.HasItems)
-{
-    <a class="nav-link dropdown-toggle" data-toggle="dropdown" href="@linkUrl">@menuItemPart.Name<b class="caret"></b></a>
-}
-else
-{
-    <a class="nav-link" href="@linkUrl">@menuItemPart.Name</a>
+    var name = (string)menuItemPart.Name;
+    if(Model.Level == 0 && Model.HasItems)
+    {
+        name += "<b class=\"caret\"></b>";
```

Comment:
Can you do an extra AppendHtml instead of concatenating the strings?

Reply:
Done. Like this change. Makes it cleaner and clearer.